buildtools/dep/dep.go

```diff
@@ -12,21 +11,21 @@ import (
 	"github.com/pkg/errors"
 )
 
-// Resolver contains both the Lockfile and Manifest information. Resolver implements golang.Resolver.
+// Resolver contains both the lockfile and Manifest information. Resolver implements golang.Resolver.
 type Resolver struct {
 	Manifest Manifest
```

Comment:
I think if the lockfile is private, the manifest should be too. It feels weird to have a struct with no exported fields, but this is actually quite idiomatic in Go.